OtripleS.Web.Api/Services/Foundations/StudentSemesterCourses/StudentSemesterCourseService.cs

```diff
@@ -40,55 +40,55 @@ public partial class StudentSemesterCourseService : IStudentSemesterCourseServic
-                    studentSemesterCourse.StudentId,
-                    studentSemesterCourse.SemesterCourseId);
+              StudentSemesterCourse maybeStudentSemesterCourse =
+                   await this.storageBroker.SelectStudentSemesterCourseByIdAsync(studentId, semesterCourse);
 
-            ValidateStorageStudentSemesterCourse(
-                maybeStudentSemesterCourse,
-                studentSemesterCourse.StudentId,
-                studentSemesterCourse.SemesterCourseId);
+              ValidateStorageStudentSemesterCourse(maybeStudentSemesterCourse, studentId, semesterCourse);
 
-            ValidateAgainstStorageStudentSemesterCourseOnModify(
-                inputStudentSemesterCourse: studentSemesterCourse,
-                storageStudentSemesterCourse: maybeStudentSemesterCourse);
+              return maybeStudentSemesterCourse;
+          });
 
-            return await this.storageBroker.UpdateStudentSemesterCourseAsync(studentSemesterCourse);
-        });
+        public ValueTask<StudentSemesterCourse> ModifyStudentSemesterCourseAsync(
+            StudentSemesterCourse studentSemesterCourse) => TryCatch(async () =>
+          {
+              ValidateStudentSemesterCourseOnModify(studentSemesterCourse);
 
-        public ValueTask<StudentSemesterCourse> RetrieveStudentSemesterCourseByIdAsync(
-            Guid studentId,
-            Guid semesterCourse) => TryCatch(async () =>
-        {
-            ValidateStudentSemesterCourseIdIsNull(studentId, semesterCourse);
+              StudentSemesterCourse maybeStudentSemesterCourse =
+                  await this.storageBroker.SelectStudentSemesterCourseByIdAsync(
+                      studentSemesterCourse.StudentId,
+                      studentSemesterCourse.SemesterCourseId);
 
-            StudentSemesterCourse maybeStudentSemesterCourse =
-               await this.storageBroker.SelectStudentSemesterCourseByIdAsync(studentId, semesterCourse);
+              ValidateStorageStudentSemesterCourse(
+                  maybeStudentSemesterCourse,
+                  studentSemesterCourse.StudentId,
+                  studentSemesterCourse.SemesterCourseId);
 
-            ValidateStorageStudentSemesterCourse(maybeStudentSemesterCourse, studentId, semesterCourse);
+              ValidateAgainstStorageStudentSemesterCourseOnModify(
+                  inputStudentSemesterCourse: studentSemesterCourse,
+                  storageStudentSemesterCourse: maybeStudentSemesterCourse);
 
-            return maybeStudentSemesterCourse;
-        });
+              return await this.storageBroker.UpdateStudentSemesterCourseAsync(studentSemesterCourse);
+          });
 
         public ValueTask<StudentSemesterCourse> RemoveStudentSemesterCourseByIdsAsync(
             Guid semesterCourseId,
             Guid studentId) => TryCatch(async () =>
-        {
-            ValidateSemesterCourseId(semesterCourseId);
-            ValidateStudentId(studentId);
+          {
```

Comment:
Why this change?
Merge latest main.

Do remember to create user branch from main.